<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Juges</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/image/image/petit logo.png</normaloff>:/image/image/petit logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#Login {
	background-image: url(:/image/image/8.jpg);
}</string>
  </property>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>6</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#b6b6b6;&quot;&gt;Password&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="login_pass">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>36</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 rgba(255, 255, 230, 148), stop:0.978261 rgba(114, 107, 92, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>6</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#9e9e9e;&quot;&gt;ID&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>951</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 rgba(173, 173, 151, 172));
background-image: url(:/image/image/1.jpg);</string>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="signup_id">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0.305, y1:0.477773, x2:1, y2:1, stop:0.0869565 rgba(127, 120, 37, 148), stop:0.978261 rgba(114, 87, 21, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="signup_mail">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>340</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 rgba(255, 255, 230, 148), stop:0.978261 rgba(114, 107, 92, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>431</width>
     <height>301</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 rgba(173, 173, 151, 172));</string>
   </property>
   <property name="title">
    <string>Espace d'inscription:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#000000;&quot;&gt;E_mail&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="signup_pass">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>270</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 rgba(255, 255, 230, 148), stop:0.978261 rgba(114, 107, 92, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#000000;&quot;&gt;Password&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_inscrire">
   <property name="geometry">
    <rect>
     <x>234</x>
     <y>410</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/image/3.jpg);</string>
   </property>
   <property name="text">
    <string>S’inscrire</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#000000;&quot;&gt;ID&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_connexion">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>36</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/image/3.jpg);</string>
   </property>
   <property name="text">
    <string>Se connecter</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="login_id">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>36</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0.305, y1:0.477773, x2:1, y2:1, stop:0.0869565 rgba(127, 120, 37, 148), stop:0.978261 rgba(114, 87, 21, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
  </widget>
  <widget class="QLabel" name="statut">
   <property name="geometry">
    <rect>
     <x>698</x>
     <y>101</y>
     <width>196</width>
     <height>46</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="statut1">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>460</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="b_recuperation">
   <property name="geometry">
    <rect>
     <x>905</x>
     <y>108</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/image/9.png);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>689</x>
     <y>80</y>
     <width>261</width>
     <height>102</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 rgba(173, 173, 151, 172));</string>
   </property>
   <property name="title">
    <string>Espace de recuperation clé</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="r_mail">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>151</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0.305, y1:0.477773, x2:1, y2:1, stop:0.0869565 rgba(127, 120, 37, 148), stop:0.978261 rgba(114, 87, 21, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
  </widget>
  <widget class="QLabel" name="objet_recup_cle">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>votre mot de passe </string>
   </property>
  </widget>
  <zorder>objet_recup_cle</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>groupBox</zorder>
  <zorder>label_6</zorder>
  <zorder>login_pass</zorder>
  <zorder>label_10</zorder>
  <zorder>signup_id</zorder>
  <zorder>signup_mail</zorder>
  <zorder>label</zorder>
  <zorder>signup_pass</zorder>
  <zorder>label_7</zorder>
  <zorder>b_inscrire</zorder>
  <zorder>label_11</zorder>
  <zorder>b_connexion</zorder>
  <zorder>login_id</zorder>
  <zorder>statut1</zorder>
  <zorder>groupBox_3</zorder>
  <zorder>b_recuperation</zorder>
  <zorder>statut</zorder>
  <zorder>r_mail</zorder>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
